### PR TITLE
feat: Bitmap type DUMP/RESTORE command support

### PR DESCRIPTION
**Pull request by poipoiPIO**
Changes requested by #2518 

## Summary
This PR extends dump and restore operations to provide support for bitmap-type objects using existing string object serialisation and deserialisation routines.

### CODE REVIEW

**Comment by git-hulk** [comment]
@poipoiPIO thanks for your contribution.

File: src/storage/rdb.cc [hunk 1]
```diff
@@ -402,7 +405,7 @@ StatusOr<int> RDB::loadRdbType() {
 }
 
 StatusOr<RedisObjValue> RDB::loadRdbObject(int type, [[maybe_unused]] const std::string &key) {
-  if (type == RDBTypeString) {
+  if (type == RDBTypeString || type == RDBTypeBitmap) {
```
Comment (git-hulk) [inline src/storage/rdb.cc]: The bitmap type is an internal represent type in Kvrocks, and it should be regarded as a string type. I think we can remove `RDBTypeBitmap`? also, the number `6` is now used by [RDB_TYPE_MODULE_PRE_GA](https://github.com/redis/redis/blob/unstable/src/rdb.h#L60C9-L60C31), reuse this number might cause the unexpected error while restoring rdb from Redis.

File: src/storage/rdb.cc [hunk 2]
```diff
@@ -728,6 +731,8 @@ Status RDB::SaveObjectType(const RedisType type) {
     robj_type = RDBTypeSet;
   } else if (type == kRedisZSet) {
     robj_type = RDBTypeZSet2;
+  } else if (type == kRedisBitmap) {
+    robj_type = RDBTypeBitmap;
```
Comment (git-hulk) [inline src/storage/rdb.cc]: As mentioned in the above comment, we can convert the robj_type to RDBTypeString here.

File: src/storage/rdb.cc (hunk 1 above)
Comment (poipoiPIO) [reply src/storage/rdb.cc]: Oh I see, thank you for the suggestion! Will do!

**Review by git-hulk — APPROVED** [review]

**Review by torwig — APPROVED** [review]

**Comment by git-hulk** [comment]
@poipoiPIO Thank you.

**Comment by poipoiPIO** [comment]
> @poipoiPIO Thank you.

Thank you for reviewing!

File: src/storage/rdb.cc [hunk 3]
```diff
@@ -781,6 +784,16 @@ Status RDB::SaveObject(const std::string &key, const RedisType type) {
     }
 
     return SaveHashObject(field_values);
+  } else if (type == kRedisBitmap) {
+    std::string value;
+    redis::Bitmap bitmap_db(storage_, ns_);
+    Config *config = storage_->GetConfig();
+    uint32_t max_btos_size = static_cast<uint32_t>(config->max_bitmap_to_string_mb) * MiB;
+    auto s = bitmap_db.GetString(ctx, key, max_btos_size, &value);
```
Comment (mapleFU) [inline src/storage/rdb.cc:792]: @git-hulk would this meet the same issue like lsb to msb?

**Pull request merged**